Source/WebCore/platform/graphics/ca/cocoa/WebTiledBackingLayer.mm

```diff
@@ -55,6 +55,12 @@ - (void)dealloc
 {
     ASSERT(!_tileController);
     _tileController = makeUnique<WebCore::TileController>(rootLayer);
+
+    // Sync the underlying layer with the controller's scale, and keep the rasterization scale the same, as PlatformCALayerCocoa does.
+    CGFloat initialScale = _tileController->contentsScale();
+    [super setContentsScale:initialScale];
```

Comment:
Are there cases where `setContentsScale:` won't get called later?

Reply:
The calling code uses the getter and only calls `setContentsScale` if the values are different.

The tile-controller self-initialises its scale value, so `setContentsScale` doesn't usually get called.